recipes/nidaqmx-python/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "nidaqmx-python" %}
+{% set version = "0.5.7" %}
+{% set sha256 = "f3d3df29ccfbb4f278648f101efde475f66a0c306fde918c06679dd323574f5b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/ni/{{ name }}/archive/{{ version }}.tar.gz
```

Comment:
FTR I deliberately switched to taking the file from github since the version on pypi was missing the license file

Reply:
You can always package the `license_file` manually:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

Reply:
thanks @synapticarbors, I was aware of that possibility but I preferred the "batteries included" approach 😉